src/__tests__/components/Login.test.js

```diff
@@ -90,4 +90,23 @@ describe("Testing Login", () => {
     const forgotPasswordButton = getByText("Tryck här");
     fireEvent.press(forgotPasswordButton);
   });
+
+  it("Trying to login with a space before  or after entering an e-mail its just takes away that space ", () => {
```

Comment:
change to "Shouldn take away space if a  user wrote one in the Login page at email field"